code/ColladaExporter.cpp

```diff
@@ -1136,21 +1136,23 @@ void ColladaExporter::WriteJointsWeightSourceNode(const size_t pIndex){
 	delete [] tempBoneWeight;
 }
 
+template <typename T>
+static void moveMatrixInArray(const aiMatrix4x4t<T> &matrix, T *outBuffer){
+	std::memcpy(outBuffer,matrix[0],16*sizeof(T));
```

Comment:
should this not be &matrix[0]?

Also wondering if perhaps a copy with two loops is better to maintain.